### PR TITLE
[SSL] More context on DCV steps and relevant parties

### PR DESCRIPTION
**Pull request by RebeccaTamachiro**
PCX-7274

### CODE REVIEW

File: content/ssl/edge-certificates/changing-dcv-method/dcv-flow.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+weight: 2
+meta:
+    description: Consider the steps that have to take place before the DCV process is completed and certificate authorities can issue SSL/TLS certificates.
+---
+
+# Domain control validation flow
+
+In order to obtain [Universal](/ssl/edge-certificates/universal-ssl/), [Advanced](/ssl/edge-certificates/advanced-certificate-manager/), and [Custom hostname](/cloudflare-for-platforms/cloudflare-for-saas/security/certificate-management/) certificates, Cloudflare partners with different publicly trusted [certificate authorities (CAs)](/ssl/reference/certificate-authorities/).
+
+However, every time a CA will issue or renew a certificate, the requester must prove that they have control over the domain. That is when the DCV process takes place, with the proof usually consisting of placing an HTTP token at a standard URL path (`/.well-known/pki-validation`), or placing a TXT record at the authoritative DNS provider.
```
Comment (marciocloudflare) [inline content/ssl/edge-certificates/changing-dcv-method/dcv-flow.md]: I think `every time a CA issues or renews` might be a more appropriate tense? Wdyt?

File: content/ssl/edge-certificates/changing-dcv-method/dcv-flow.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+* The CA that processes the request.
+
+## Steps in the process
+
+In summary, five steps have to succeed after Cloudflare requests a CA to issue or renew a certificate:
+
+1. Cloudflare receives the DCV tokens from the CA.
+2. Cloudflare either places the tokens on your behalf ([Full DNS setup](/dns/zone-setups/full-setup/), [Delegated DCV](/ssl/edge-certificates/changing-dcv-method/methods/delegated-dcv/)), or makes the tokens available for you to place them.
+2. Cloudflare polls the validation URLs to check for the tokens.
+3. After Cloudflare can see that the tokens are placed via multiple DNS resolvers, the CA is asked to check as well.
```
Comment (marciocloudflare) [inline content/ssl/edge-certificates/changing-dcv-method/dcv-flow.md]:
```suggestion
3. After Cloudflare can confirm that the tokens are placed via multiple DNS resolvers, the CA is asked to check as well.
```

Comment (marciocloudflare) [reply content/ssl/edge-certificates/changing-dcv-method/dcv-flow.md]: From an inclusivity standpoint, it's best to avoid the verb "to see".

**Review by marciocloudflare — APPROVED** [review]

File: content/ssl/edge-certificates/changing-dcv-method/dcv-flow.md (hunk 1 above)
Comment (RebeccaTamachiro) [reply content/ssl/edge-certificates/changing-dcv-method/dcv-flow.md]: Thanks for the callout. `will issue` sounded okay to me, but have changed it to `every time a CA is requested to issue or renew`. A bit of a technicality in this case, but only to keep very consistent with the order in which the steps occur. Certificate is not actually issued nor renewed if the process is not successful.

**Pull request merged**